lib/twemproxy_exporter/counter.rb

```diff
@@ -0,0 +1,19 @@
+module TwemproxyExporter
+  class Counter
+    def initialize(registry, name, desc)
+      @counter = Prometheus::Client::Counter.new(name, desc)
+      registry.register(@counter)
+      @last = 0
+    end
+
+    def count(value, labels = {})
```

Comment:
Yeah - this is a little weird.  Prometheus itself seems to handle things as I'd expect, but the client libraries don't ([Ruby example](https://github.com/prometheus/client_ruby/blob/master/lib/prometheus/client/counter.rb#L13))...  I think the current logic is what's needed to handle a Twemproxy counter reset the same way Prometheus would.